Mathlib/CategoryTheory/Grothendieck.lean

```diff
@@ -149,6 +161,101 @@ def forget : Grothendieck F ⥤ C where
 
 end
 
+section
+
+variable {G : C ⥤ Cat}
+
+/-- The Grothendieck construction is functorial: a natural transformation `α : F ⟶ G` induces
+a functor `Grothendieck.map : Grothendieck F ⥤ Grothendieck G`.
+-/
+@[simps!]
+def map (α : F ⟶ G) : Grothendieck F ⥤ Grothendieck G where
+  obj X :=
+  { base := X.base
+    fiber := (α.app X.base).obj X.fiber }
+  map {X Y} f :=
+  { base := f.base
+    fiber := (eqToHom (α.naturality f.base).symm).app X.fiber ≫ (α.app Y.base).map f.fiber }
+  map_id X := by
+    simp only [Cat.comp_obj, id_fiber', eqToHom_map]
+    congr 1
+    rw [eqToHom_app, eqToHom_trans]
+  map_comp {X Y Z} f g := by
+    dsimp
+    congr 1
+    simp only [comp_fiber' f g, ← Category.assoc, Functor.map_comp, eqToHom_map]
+    apply congrFun (congrArg (. ≫ .) ?_) ((α.app Z.base).map g.fiber)
+    have H := Functor.congr_hom (α.naturality g.base).symm f.fiber
+    erw [H, eqToHom_app, eqToHom_app]
```

Comment:
It's this `erw` that's making the proof slow. Maybe there are some `simp` lemmas missing?